index.html

```diff
@@ -227,8 +215,29 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
           </ul>
       </div>
     </section>
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+    <form action="" class="contact-form">
+      <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+      <input type="text" class="contact-input" placeholder="Email address" required>
+      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500">Enter text here</textarea>
+      <button class="contact-btn" type="button">Get in touch</button>
+    </form>
```

Comment:
- [ ] You have shown a high level of understanding of the concepts by implementing the contact form 👏 👏 . However, the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#htmlcss-requirements) state that, the user must be able to submit the form and collect data by using the [Formspree service](https://formspree.io/html)☺️.  Kindly ensure to enable form submission.
- **Hint:** Consider using the correct attributes in the `<form>`. ☺️